ui/src/main/kotlin/edu/artic/view/ArticMainAppBarLayout.kt

```diff
@@ -20,8 +18,8 @@ import kotlinx.android.synthetic.main.view_app_bar_layout.view.*
  * this app.
  */
 class ArticMainAppBarLayout(context: Context, attrs: AttributeSet? = null) : AppBarLayout(context, attrs) {
-    private val disposeBag : DisposeBag = DisposeBag()
-    private var clickConsumer : Consumer<Unit>? = null
+    private val disposeBag: DisposeBag = DisposeBag()
```

Comment:
where is the disposebag clear? I know I created it, but can you add disposebag.clear remove from window?

Reply:
It's inside `onDetachedFromWindow`.